lib/model-utils.js

```diff
@@ -88,6 +88,7 @@ const operators = {
   ilike: 'ILIKE',
   nilike: 'NOT ILIKE',
   regexp: 'REGEXP',
+  contains: '@>',
```

Comment:
Please note that the operator needs to be supported by other LoopBack db connectors beyond Postgres. Do we have some ideas about the feasibility of other DBs?

Reply:
@raymondfeng This feature is so much needed. We literally had to hack our way into Loopback as we needed it extensively. Please figure out a way to get this merged.

Reply:
@raymondfeng currently i have added it only for postgres db, not sure how we may implement it for other db connectors as well, i may need to look into this..

Reply:
I agree with @raymondfeng that we must keep other connectors in mind when making any changes in juggler.

Is it possible to implement array coercion of `contains` filter inside the postgresql connector?

In MongoDB, we have the concept of extended operators, see line 507 later in this file.

https://github.com/strongloop/loopback-datasource-juggler/blob/d953eabe30d793642b9268014efc5f4da4633e29/lib/model-utils.js#L507

Can we perhaps use the same mechanism for PostgreSQL?

Reply:
I need to check into this @bajtos will inform you after analysis.

Reply:
@bajtos you are right after using allowExtendedOperators option within postgres connector model, I am able to handle the case for array field filter.